src/cntk-train/src/main/python/CNTKLearner.py

```diff
@@ -8,11 +8,31 @@
 
 from mmlspark._CNTKLearner import _CNTKLearner
 from mmlspark.CNTKModel import CNTKModel as CNTKmod
-from pyspark.ml.common import inherit_doc
+from pyspark import SparkContext
+from pyspark.ml.common import inherit_doc, _java2py
 
 @inherit_doc
 class CNTKLearner(_CNTKLearner):
     """
+    Transforms the params from java object, with special case for byte array
+
+    Args:
+        java_model (CNTKModel): The python CNTKModel to transfer parameters to from java
+    """
+    def _custom_transfer_params_from_java(self, model):
+        for param in model.params:
+            if model._java_obj.hasParam(param.name):
+                java_param = model._java_obj.getParam(param.name)
+                if model._java_obj.isSet(java_param):
+                    param_val = model._java_obj.getOrDefault(java_param)
+                    if isinstance(param_val, (bytearray, bytes)):
```

Comment:
See my other comment on Mark's PR #98 - checking ``isinstance`` is generally not preferred in Python. 
Also, does this code work in both 2.7 and 3.6?